apps/indexer/src/chain/dataStore.ts

```diff
@@ -6,6 +7,7 @@ import { dataFolderPath } from "@src/shared/constants";
 import { bytesToHumanReadableSize } from "@src/shared/utils/files";
 
 const LevelNotFoundCode = "LEVEL_NOT_FOUND";
+const logger = LoggerService.forContext("dataStore");
```

Comment:
let's call all of them in the same notation pls
```suggestion
const logger = LoggerService.forContext("DataStore");
```